src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -1491,6 +1491,117 @@ def cleanse_crash(self,
       return result
 
 
+class AndroidEmulatorLibFuzzerRunner(AndroidLibFuzzerRunner):
+  """Android emulator libFuzzer runner."""
+
+  # pylint: disable=super-init-not-called
+  def __init__(self, executable_path, build_directory, default_args=None):
+    """Inits the AndroidEmulatorLibFuzzerRunner.
+
+    Args:
+      executable_path: Path to the fuzzer executable.
+      build_directory: A MinijailChroot.
+      default_args: Default arguments to always pass to the fuzzer.
+    """
+    self.exec_path = executable_path
+    self.build_dir = build_directory
+    self.def_args = default_args
+    # We don't call super().__init__() here since we need to wait until
```

Comment:
What's the reason why we need to do this? 

Can we refactor the base class instead to avoid doing this?

Reply:
I've change this to just call the required functionality (copying files to the device) directly.